### PR TITLE
Proper naming for no-op / doNothing functions

### PR DESCRIPTION
**Pull request by kleril**
Mostly pruning of low-hanging fruit.

Stuff that wasn't immediately obvious and self-explanatory that I haven't touched, and could use a more experienced set of eyes:

room.c / play.c / functions.h
- func_80095AA0

kanfont.c / message.c / functions.h
- func_8006EE50

synthesis.c
- the entire block between lines 340 & 400

general.c
- func_800F3138
- func_800F3140

### CODE REVIEW

File: src/overlays/actors/ovl_Bg_Spot11_Oasis/z_bg_spot11_oasis.c [hunk 1]
```diff
@@ -106,19 +106,19 @@ void func_808B29E0(BgSpot11Oasis* this) {
...
         Actor_Spawn(&play->actorCtx, play, ACTOR_EN_ELF, this->actor.world.pos.x, this->actor.world.pos.y + 40.0f,
                     this->actor.world.pos.z, 0, 0, 0, FAIRY_SPAWNER);
         Sfx_PlaySfxCentered(NA_SE_SY_CORRECT_CHIME);
     }
     func_808B27F0(play, this->actor.world.pos.y);
 }
 
-void func_808B2AA8(BgSpot11Oasis* this) {
-    this->actionFunc = func_808B2AB8;
+void BgSpot11Oasis_SetActionToNothing(BgSpot11Oasis* this) {
```
Comment (cadmic) [inline src/overlays/actors/ovl_Bg_Spot11_Oasis/z_bg_spot11_oasis.c:117]: The convention seems to be `<Actor>_Setup<Action>`, so `BgSpot11Oasis_SetupDoNothing` (like `BgJya1flift_SetupDoNothing`). You could probably apply this to other functions in this PR too, like `func_808B3274`

File: src/overlays/actors/ovl_En_Elf/z_en_elf.c [hunk 2]
```diff
@@ -424,7 +424,8 @@ void EnElf_Init(Actor* thisx, PlayState* play) {
     }
 }
 
-void func_80A0299C(EnElf* this, s32 arg1) {
+//TODO: Technically a noop, but the argument and values passed to it suggest it may have done *something*.
```
Comment (cadmic) [inline src/overlays/actors/ovl_En_Elf/z_en_elf.c:427]: Nit: add space between `//` and `TODO`.

Although, maybe this function is related to `func_80A029A8` below? In that case I think it's probably to name these two functions together somehow (though in another PR probably)

**Review by cadmic — COMMENTED** [review]
Seems like CI is complaining about formatting. Could you run `./format.py -j<N>` to reformat the files?

**Review by cadmic — APPROVED** [review]

File: src/overlays/actors/ovl_player_actor/z_player.c [hunk 3]
```diff
@@ -9922,11 +9922,11 @@ static ColliderQuadInit D_808546A0 = {
...
-void func_8084663C(Actor* thisx, PlayState* play) {
+void Player_DoNothing(Actor* thisx, PlayState* play) {
 }
 
-void func_80846648(PlayState* play, Player* this) {
-    this->actor.update = func_8084663C;
+void Player_SetupDoNothing(PlayState* play, Player* this) {
+    this->actor.update = Player_DoNothing;
     this->actor.draw = NULL;
 }
```
Comment (fig02) [inline src/overlays/actors/ovl_player_actor/z_player.c:9931]: These functions in player specifically are "initMode" functions, that control how player "loads into" the world. I think this should be documented with the rest of the initModes. I would personally revert all these changes in player.

**Review by fig02 — COMMENTED** [review]
In general I'm not a huge fan of this approach-- I think these functions should be named by the person documenting the individual system or actor. Not all empty functions are made equal, and they still deserve context dependent names.

But, this isn't stopping someone from changing the names later. There are a couple that I think should be reverted though

File: src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.c [hunk 4]
```diff
@@ -228,7 +228,7 @@ void ObjWarp2block_Destroy(Actor* thisx, PlayState* play) {
     }
 }
 
-void ObjWarp2block_SetInactive(ObjWarp2block* this) {
+void ObjWarp2block_SetupDoNothing(ObjWarp2block* this) {
     this->actionFunc = ObjWarp2block_DoNothing;
     this->dyna.actor.draw = NULL;
```
Comment (Dragorn421) [inline src/overlays/actors/ovl_Obj_Warp2block/z_obj_warp2block.c:233]: this does something x4

File: src/overlays/actors/ovl_Bg_Spot12_Gate/z_bg_spot12_gate.c [hunk 5]
```diff
@@ -129,12 +129,12 @@ void func_808B318C(BgSpot12Gate* this, PlayState* play) {
     }
 }
 
-void func_808B3274(BgSpot12Gate* this) {
-    this->actionFunc = func_808B3298;
+void BgSpot12Gate_SetupDoNothing(BgSpot12Gate* this) {
+    this->actionFunc = BgSpot12Gate_DoNothing;
     this->dyna.actor.world.pos.y = this->dyna.actor.home.pos.y + 200.0f;
```
[the remark this answers is not in the document]
Comment (cadmic) [reply src/overlays/actors/ovl_Bg_Spot12_Gate/z_bg_spot12_gate.c:134]: I think `SetupDoNothing` is a fine name, it does whatever setup is needed for the actor to then do nothing

**Comment by fig02** [comment]
Closing due to inactivity. 
Feel free to reopen this PR if you get back around to oot things and want to maintain the PR again

**Pull request closed without merging**